PoshCodex/Source/Private/Write-Completion.ps1

```diff
@@ -25,6 +25,6 @@ function Write-Completion {
 		[Microsoft.PowerShell.PSConsoleReadLine]::Insert($completion)
 	}
 	else {
-		Write-Output 'Response returned by API is null! It could be an internal error or the model is not installed properly hrough Ollama. Please fix and try again.' -ForegroundColor Red
+		Write-Host 'Response returned by API is null! It could be an internal error or the model is not installed properly through Ollama. Please fix and try again.' -ForegroundColor Red
```

Comment:
@rishi255 I think that it would be better to throw exception, instead of *fake* error message. For the moment I've just replaced `Write-Output` with `Write-Host` since the former does not support color output.